clustering/common/src/main/java/org/jboss/as/clustering/controller/AddStepHandler.java

```diff
@@ -94,7 +94,8 @@ protected void recordCapabilitiesAndRequirements(OperationContext context, Model
         for (Capability capability : this.descriptor.getCapabilities()) {
             context.registerCapability(capability.getRuntimeCapability(address), null);
         }
-        super.recordCapabilitiesAndRequirements(context, operation, resource);
+        //don't call super as we already register everything here
+        //super.recordCapabilitiesAndRequirements(context, operation, resource);
```

Comment:
This implementation isn't doing everything the superclass does, i.e. registering requirements from attributes. Since this class doesn't pass any capabilities to the superclass constructor, how does what it's doing conflict with what the superclass does?